roles/openshift_logging/library/openshift_logging_facts.py

```diff
@@ -77,9 +77,9 @@ def oc_command(self, sub, kind, namespace=None, name=None, add_options=None):
             process = Popen(cmd, stdout=PIPE, stderr=PIPE)   # noqa: F405
             out, err = process.communicate(cmd)
```

Comment:
If you are only going to use `err` as a string, why not just do
```python
    err = str(err)
```
?

Reply:
Yes, did that instead but with `decode`. 🙂